profiles/apparmor/template.go

```diff
@@ -17,6 +17,12 @@ profile {{.Name}} flags=(attach_disconnected,mediate_deleted) {
   capability,
   file,
   umount,
+{{if ge .Version 208096}}
+{{/* Allow 'docker kill' to actually send signals to container processes. */}}
+  signal (receive) peer=unconfined,
```

Comment:
Just to note, this is the first time that the daemon assumes it runs as "unconfined". I don't run it as that profile, so I had to tweak it.

Since this is dynamically generated, maybe we could replace this with:

```
signal (receive) peer={{.DaemonProfile}}
```

Then fill that value in with whatever's in `/proc/self/attr/current`

Reply:
There is a self keyword or something I think

Reply:
I will take a look into whether there's a self keyword or if we'd need to actually read `attr/current`. I completely forgot that we have a daemon profile.

Reply:
ah yeah omg that daemon profile is bad but regardless... lgtm :)